packages/components/psammead-navigation/src/index.jsx

```diff
@@ -52,13 +44,12 @@ const ListItemBorder = `
   left: 0;
   right: 0;
   bottom: 0;
-  border-bottom: ${GEL_SPACING_HLF} solid ${C_WHITE};
 `;
 
 const StyledLink = styled.a`
   ${({ script }) => script && getPica(script)};
-  ${({ service }) => getSansRegular(service)}
-  color: ${C_GHOST};
+  ${({ service }) => getSansRegular(service)};
+  ${({ brandForegroundColour }) => `color: ${brandForegroundColour};`}
```

Comment:
Same question re defaults

Reply:
No default for this, requires a value to be passed in as a prop.